lib/hsa/kernels/rfi_chime_input_sum.cl

```diff
@@ -31,18 +33,21 @@ rfi_chime_inputsum(
     short lx_size = get_local_size(0);
     //Declare Local Memory
     __local float sq_power_across_input[256];
+    __local float var_across_input[256];
     // Compute index in input array
     uint base_index = gx + gy * num_elements + gz * num_elements * gy_size;
     sq_power_across_input[lx] = input_mask[lx]*input[base_index];
     // Partial sum if more than 256 inputs
     for(int i = 1; i < num_elements/lx_size; i++) {
         sq_power_across_input[lx] += input_mask[lx + i * lx_size] * input[base_index + i * lx_size];
+        var_across_input[lx] += input_mask[lx + i * lx_size] * input_var[base_index + i * lx_size];
```

Comment:
`var_across_input[lx]` is never initialized to a starting value, so you are just adding to whatever is in memory.  You need a line like the `sq_power_across_input[lx] = input_mask[lx]*input[base_index];` one above.

Reply:
Ah thanks, that'll be it. I never noticed that